### PR TITLE
[CP 2.5] Bump rubygem-foreman_rh_cloud to 4.0.22-1

### PR DESCRIPTION
**Pull request by laviro**

### CODE REVIEW

File: packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec [hunk 1]
```diff
@@ -5,10 +5,10 @@
 
 %global gem_name foreman_rh_cloud
 %global plugin_name rh_cloud
-%global foreman_min_version 2.3
+%global foreman_min_version 2.4
```
Comment (evgeni) [inline packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec:8]: The version in develop has `foreman_min_version 2.5` -- what's correct?

Comment (evgeni) [reply packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec:8]: This is not what @shimstein said in https://github.com/theforeman/foreman-packaging/pull/6750#discussion_r646116639

I don't mind either way, but can you two please agree ;)

Comment (laviro) [reply packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec:8]: I guess it's not the best idea for me to do packaging PR in the evenings :P
updating 👍🏼

**Comment by laviro** [comment]
Done @evgeni ! good eye :)

**Review by evgeni — APPROVED** [review]

**Pull request merged**